packages/interactivity/src/proxies/context.ts

```diff
@@ -0,0 +1,69 @@
+// TODO: Use the proxy registry to avoid multiple proxies on the same object.
```

Comment:
What is this?

Reply:
It's just a reminder of what I mentioned above in the description:

> It's a bit tricky to merge the state proxies logic with the context proxies one, so we could address that in a future PR.

Reply:
Ok, thanks! But, to be honest, I wouldn't note that in the code…

Reply:
I've moved it above `contextHandlers`.